test/new-e2e/examples/agentenv_file_permissions_test.go

```diff
@@ -90,8 +90,8 @@ func (v *filePermissionsTestSuite) TestUserGroupPermissions() {
 
 func (v *filePermissionsTestSuite) TestSecretsPermissions() {
 	files := []agentparams.Option{
-		agentparams.WithFileWithPermissions("/tmp/secrets", "typical secret config", false, secrets.WithUnixSecretPermissions(false)),
-		agentparams.WithFileWithPermissions(`/tmp/secrets_root_group`, "secret config for security agent", false, secrets.WithUnixSecretPermissions(true)),
+		secretsutils.WithUnixSetupScript("/tmp/secrets", false),
+		secretsutils.WithUnixSetupScript("/tmp/secrets_root_group", false),
```

Comment:
This line seems to be changing: before the `allowGroupExec` bool would be `true`, but now it's `false`. It may not matter, especially if the test still passes, but just wanted to point it out and make sure it was intentional.